listeners/measured.go

```diff
@@ -27,14 +27,14 @@ func (l *stateAwareMeasuredListener) Accept() (c net.Conn, err error) {
 	sac, _ := c.(*measured.Conn).Conn.(WrapConnEmbeddable)
 	return &wrapMeasuredConn{
 		WrapConnEmbeddable: sac,
-		Conn:               *c.(*measured.Conn),
+		Conn:               c.(*measured.Conn),
```

Comment:
This was the only thing the race detector complained about.  What's another example of us using this pattern?